### PR TITLE
Get submodules of multi module project

### PR DESCRIPTION
**Pull request by robstoll**
based on #55 please merge it first (this way you see what was actually added with this pull request).

I added the method getSubmodules(Gav) to ProjectRepository which returns all submodules of the project behind the given gav and filters those which are not part of the repository.

### CODE REVIEW

File: pom-explorer-core/src/main/java/fr/lteconsulting/pomexplorer/Project.java [hunk 1]
```diff
@@ -773,6 +774,36 @@ private PropertyLocation getPropertyDefinition( Log log, String propertyName, bo
 		return null;
 	}
 
+	/**
+	 * @return All submodules of this project or an empty list if it does not have any.
+	 */
+	public Stream<Project> getSubmodules()
```
Comment (ltearno) [inline pom-explorer-core/src/main/java/fr/lteconsulting/pomexplorer/Project.java]: Everywhere else we use List<>, let's keep on with that.
Better to have the same kind of API everywhere

**Comment by ltearno** [comment]
Maybe we could have a voice discussion about what is your mid-term view of what you want to add to PomExplorer, so that we can agree beforehand on bigger impacts ?

File: pom-explorer-core/src/main/java/fr/lteconsulting/pomexplorer/Project.java (hunk 1 above)
Comment (robstoll) [reply pom-explorer-core/src/main/java/fr/lteconsulting/pomexplorer/Project.java]: Fair enough, I am going to change that

**Comment by robstoll** [comment]
Sure why not. I write you an email

**Pull request merged**